test/unit/preloaded-vl-proza-message.test.html

```diff
@@ -58,8 +59,10 @@
         test('toont een bericht', () => {
             const proza = fixture('vl-proza-preloaded-messages-fixture');
             return fetchMock.flush(true).then(() => {
-                prozaMessageContent(proza.querySelector('#message-1')).should.equal('foobar');
-                prozaMessageContent(proza.querySelector('#message-2')).should.equal('foofoo');
+                return awaitUntil(() => {
+                    return prozaMessageContent(proza.querySelector('#message-1')) === 'foobar' &&
+                        prozaMessageContent(proza.querySelector('#message-2')) === 'foofoo';
+                });
```

Comment:
Waarom is er een await nodig als de fetch wait for body true is?

Reply:
Ja, ik vond het ook vreemd, maar het leek alsof er nog ergens asynchroniteit over bleef aangezien er toch soms falende testen waren zonder die awaitUntil. Misschien door het chainen van promises, dat hij toch ergens nog op een async manier bepaalde stukjes verwerkt.